usersync/cookie_test.go

```diff
@@ -130,7 +125,7 @@ func TestParseCorruptedCookieJSON(t *testing.T) {
 }
 
 func TestParseNilSyncMap(t *testing.T) {
-	cookieJSON := "{\"bday\":123,\"optout\":true}"
+	cookieJSON := "{\"optout\":123}"
```

Comment:
Should it be "true"?
```go 
"{\"optout\":true}"
```

Reply:
This line looks like this because we want the json to be malformed. So that's why there's a invalid value here.

Reply:
I see, yes, `malformed` is more descriptive.